docs/content/docs/next-js.md

```diff
@@ -0,0 +1,142 @@
+---
+summary: Setup a Monorepo with AdonisJS, Next.js and Tuyau
+---
+
+In this guide, we'll be setting up a monorepo with AdonisJS, Next.js and Tuyau.
+
+# Monorepo
+
+We'll be using `pnpm` to manage the dependencies in our monorepo.
+
+Let's create an empty folder called `acme` and initialize `pnpm`:
+
+```sh
+pnpm init
+```
+
+As a result, we'll have a `acme` folder with the following structure:
+
+```
+acme
+├── package.json
+```
+
+Now we need to tell `pnpm` where our apps are located in the folder and for that we need to add a [`pnpm-workspace.yaml`](https://pnpm.io/workspaces) file as `pnpm` do not support the `workspaces` field in `package.json`:
+
+```yaml
+# pnpm-workspace.yaml
+packages:
+    - apps
```

Comment:
Shouldn't it be `apps/*`? Or is it the same thing? I personally always use `apps/*`

Reply:
No you are right, it should be `apps/*`